12_Bombs_Away/python/bombs_away.py

```diff
@@ -0,0 +1,191 @@
+"""
+Bombs away
+
+Ported from BASIC to Python3 by Bernard Cooke (bernardcooke53)
+Tested with Python 3.8.10, formatted with Black and type checked with mypy.
+"""
+import random
+from typing import Iterable
+
+
+def _stdin_choice(prompt: str, *, choices: Iterable[str]) -> str:
+    ret = input(prompt)
+    while ret not in choices:
+        print("TRY AGAIN...")
+        ret = input(prompt)
+    return ret
+
+
+def player_survived() -> None:
+    print("YOU MADE IT THROUGH TREMENDOUS FLAK!!")
+
+
+def player_death() -> None:
+    print("* * * * BOOM * * * *")
+    print("YOU HAVE BEEN SHOT DOWN.....")
+    print("DEARLY BELOVED, WE ARE GATHERED HERE TODAY TO PAY OUR")
+    print("LAST TRIBUTE...")
+
+
+def mission_success() -> None:
+    print(f"DIRECT HIT!!!! {int(100 * random.random())} KILLED.")
+    print("MISSION SUCCESSFUL.")
+
+
+def death_with_chance(p_death: float) -> bool:
+    """
+    Takes a float between 0 and 1 and returns a boolean
+    if the player has survived (based on random chance)
+
+    Returns True if death, False if survived
+    """
+    return p_death > random.random()
+
+
+def commence_non_kamikazi_attack() -> None:
+    while True:
+        try:
+            nmissions = int(input("HOW MANY MISSIONS HAVE YOU FLOWN? "))
+
+            while nmissions >= 160:
+                print("MISSIONS, NOT MILES...")
+                print("150 MISSIONS IS HIGH EVEN FOR OLD-TIMERS")
+                nmissions = int(input("NOW THEN, HOW MANY MISSIONS HAVE YOU FLOWN? "))
+                break
```

Comment:
With this `break` where it is now it looks like this inner `while` loop only runs once.  Did you mean for this `break` to be dedented one level?